scripts/build-post-list.js

```diff
@@ -57,6 +57,12 @@ function walkDirectories(directories, result, sectionWeight = 0, sectionTitle) {
         details.sectionTitle = sectionTitle
         details.isIndex = fileName.endsWith('/index.md')
         details.slug = details.isIndex ? sectionSlug : slug.replace(/\.md$/, '')
+        if(details.slug.includes('/specifications/') && !details.title) {
+          details.title = capitalize(basename(data.slug))
+          if(basename(data.slug).startsWith('v')) {
+            details.title = capitalize(basename(data.slug).slice(1))
+          }
```

Comment:
```suggestion
          if(basename(data.slug).startsWith('v')) {
            details.title = capitalize(basename(data.slug).slice(1));
          } else {
            details.title = capitalize(basename(data.slug));
          }
```

Reply:
@aayushmau5 Please only add this `else` statement :)

Reply:
Done. I was trying to be clever earlier :sweat_smile:

Reply:
Don't be so hard on yourself. Really, well done job! I love it! :)